R/validate_model_data.R

```diff
@@ -112,6 +116,19 @@ validate_model_data <- function(hub_path, file_path, round_id_col = NULL,
     return(checks)
   }
 
```

Comment:
Adding the extraction of `derived_task_ids` here so that it follows checks on `round_id`